src/lib.rs

```diff
@@ -25,7 +25,7 @@
 #![deny(non_camel_case_types)]
 #![deny(non_snake_case)]
 #![deny(unused_mut)]
-#![deny(missing_docs)]
+//#![deny(missing_docs)]
```

Comment:
I hope you didn't intend to land this :)

Reply:
removed, and added doc where missing in 8bf2f9ed2fd5f6be0693b502e9a5351edef300d4